resources/views/layouts/app.blade.php

```diff
@@ -71,7 +71,7 @@ class="@if(Settings::usesDarkTheme()) dark @endif @if(Settings::usesCompactTable
                             target="_blank"
                         >
                             <x-ark-icon
-                                name="brands.coingecko"
+                                name="app-coin-gecko"
```

Comment:
Food for thought, I can't find anywhere that hyphenates coingecko. on their site they do capitalise them (CoinGecko) but that's all. I'll approve as it's a trivial thing

Reply:
changing it to `coingecko` to keep it consistent with other uses too